### PR TITLE
Changed all displayed instances of alert to scheduled query

### PR DESCRIPTION
**Pull request by qv-pat**
https://analyticsmd.atlassian.net/browse/ENG-11084
https://analyticsmd.atlassian.net/browse/ENG-11052

### CODE REVIEW

**Comment by yuguang** [comment]
@qv-pat Does edit scheduled query have the same interface as create?

File: superset/connectors/sqla/models.py [hunk 1]
```diff
@@ -255,7 +255,7 @@ class TimePeriod(enum.Enum):
     day = 60 * 24
 
 
-class Alert(Model):
+class Alert(Model, AuditMixinNullable):
```
Comment (yuguang) [inline superset/connectors/sqla/models.py:258]: @qv-pat Does the AuditMixinNullable add in the created by, changed on, created on fields?

Comment (qv-pat) [reply superset/connectors/sqla/models.py:258]: Yup

**Comment by qv-pat** [comment]
The edit page does not have all the options that the create page has at the moment. Flask automatically generates an edit page for each model with whatever columns you specify. Creating a separate edit page will probably take some more work. I know there's a ticket for it, but it'll likely be in a separate PR.

**Review by yuguang — APPROVED** [review]
LGTM

**Comment by yuguang** [comment]
@qv-pat You can merge when ready.

**Pull request merged**